test/upsert/autospill/03-rocksdb.td

```diff
@@ -34,7 +35,7 @@ fish:AREALLYBIGFISHAREALLYBIGFISHAREALLYBIGFISHAREALLYBIGFISH
   WHERE t.name = 'autospill_tbl'
   GROUP BY t.name
   ORDER BY t.name
-true true 3
+true 3
```

Comment:
It looks like the checks below might also fail. In particular, the next one fetches the current value of `bytes_indexed` and then compares it later to see if it got smaller. If the value is already 0 here, then it won't be able to get smaller (assuming it's unsigned).